planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
 <!-- issue title -->
 
-## Site Title
+## Site Title (milestone)
```

Comment:
What do you mean by "milestone"?

We have milestones such as : must have, should have, could have ..
Do you want to add more?

Reply:
Yeah while doing each part of the website we can  make it as one of the mile stone...eg if your doing content part then content is a mile stone according to my understanding.

Reply:
Didi you read this 
https://github.com/HackYourFutureBelgium/agile-development/blob/master/deliverables/all-about-trees/issues.md ?

It's okay for me to create a new milestones, I am just trying to understand its functionallity

Reply:
Yeah ... must have, should have and could have are also mile stone. Can you put it in the discussion whether intro, title can be a milestone

Reply:
I think it's a good idea to have milestones like Title, main text... Please say @Carsiti  to add them. 
I added for my Title issue

Reply:
yeah perfect